distribution.pm

```diff
@@ -104,7 +104,7 @@ sub script_run {
 
     testapi::type_string "$name";
     if ($wait > 0) {
-        my $str = bmwqemu::random_string(5);
+        my $str = bmwqemu::hashed_string("SR$name$wait");
```

Comment:
You could generate the prefix with
```
(caller(0))[3] =~ s/([^:]*::\b|(?<=_))(\w)([^_]*_?)/\u$2/rg
```
… but this would be the road to madness, right?